src/gcp/storage.ts

```diff
@@ -158,6 +158,7 @@ export async function upload(
     method: "PUT",
     path: url.pathname,
     queryParams: url.searchParams,
+    responseType: "xml",
```

Comment:
Forgive me a silly question: could we not just add `accept: application/json` to get JSON responses from this API? Or does it exclusively respond in XML?

It feels a little weird to add support for XML for a single place - not that it's _wrong_, but _everywhere else_ returns JSON, so it's an outlier...

Reply:
It's strictly an XML endpoint we are using since the URL returned by GCF when we call "generateUploadURL" works against it:

https://cloud.google.com/storage/docs/xml-api/overview

We _could_ do some work to move everything to JSON API

https://cloud.google.com/storage/docs/json_api/v1/buckets/patch

But that's gonna be a lot more work than this.

Reply:
(Looks like XML is the encoding of choice over JSON in the storage API world)

Reply:
whoops more correct answer - signed URL only supports XML API:

https://cloud.google.com/storage/docs/access-control/signed-urls